zebedee-cms/src/main/java/com/github/onsdigital/zebedee/api/Collection.java

```diff
@@ -284,6 +284,8 @@ public boolean deleteCollection(HttpServletRequest request, HttpServletResponse
             throw new NotFoundException("The collection you are trying to delete was not found");
         }
 
+        //soft-delete any interactives
```

Comment:
I'd just be careful not to commit this as-is. Could be confused as a description of the following `deleteCollection(…)` method call. Consider `TODO`

Reply:
oops - this was left-over accidentally - i didnt realise you can only delete a collection when its empty 🤠